wp-content/themes/chrisdinh-photography/resources/views/front-page.blade.php

```diff
@@ -8,7 +12,12 @@
   @endwhile
 
   @unless ( empty($slides) && empty($sliderSettings) )
+  <pre>
```

Comment:
@jonathanmdinh remove this dump please

Reply:
removed dump in commite 35e4be2e36a8df541fc5490c5d11515040dc0b42